appcelerator/titanium_mobile_windows
# [TIMOB-25433] Add "run-cmake" on missing cmake project

[TIMOB-25433](https://jira.appcelerator.org/browse/TIMOB-25433)

Add `--run-cmake` automatically when cmake directory does not exist.

How to test:

1. Create module project
2. cd `windows/`
3. Remove `Windows10.ARM/` and `Windows10.Win32/` directory
4. `appc ti build -p windows -l trace`
5. cmake projects should be re-created and module is built
